flink-libraries/flink-table/src/main/scala/org/apache/flink/table/plan/nodes/datastream/DataStreamOverAggregate.scala

```diff
@@ -129,11 +131,17 @@ class DataStreamOverAggregate(
             generator,
             inputDS,
             isRowTimeType = false,
-            isRowsClause = overWindow.isRows)
+            isRowsClause = overWindow.isRows,
+            consumeRetraction)
         } else if (
           overWindow.lowerBound.isPreceding && !overWindow.lowerBound.isUnbounded &&
             overWindow.upperBound.isCurrentRow) {
           // bounded OVER window
+          if (consumeRetraction) {
+            throw new TableException(
+              "Retraction for bounded over window is not supported yet. Note: Currently, bounded " +
```

Comment:
1. Retraction for -> on
2. groupby -> groupBy
3. keep the term consistent either choosing 'OVER' or 'over' in the same context.